python/frontend/cirrus/cirrus/cv_utils.py

```diff
@@ -0,0 +1,38 @@
+def create_test_train_sets(num_sets, k = 5):
+    train_sets = []
+    test_sets = []
+    per_set = num_sets // k
+    remainder = num_sets - (per_set * k)
+    total_sets = [[i * per_set, (i + 1) * per_set - 1] for i in range(k)]
+    total_sets[len(total_sets) - 1][1] += remainder
+    print(total_sets)
```

Comment:
Prints of lists are not useful without other descriptions. Also, why are we not using proper logs (see automate.py).